src/tests/test_e2e_install.py

```diff
@@ -37,6 +38,47 @@ def test_olm_operator(self, cluster, olm_operators):
         cluster.prepare_for_installation()
         cluster.start_install_and_wait_for_installed(fall_on_pending_status=True)
 
+    @JunitTestSuite()
+    def test_mce_storage_post(self):
+        config.load_kube_config()
+
+        # get agent service config
+        asc_list = client.CustomObjectsApi().list_cluster_custom_object(
+            group="agent-install.openshift.io", version="v1beta1", plural="agentserviceconfigs"
+        )
+        assert "items" in asc_list
+        assert len(asc_list["items"]) == 1, "Expected to have only one agentserviceconfig resource"
+        agent_service_config = asc_list["items"][0]
+
+        # get storage classes
+        storage_classes_list = client.StorageV1Api().list_storage_class()
+        assert len(storage_classes_list.items) > 0, "Expected storage class list to have one item or more"
+        storage_classes = [storage_class.metadata.name for storage_class in storage_classes_list.items]
+
+        # each storage pvc should have an existing storage class
+        storageClaims = ["databaseStorage", "filesystemStorage", "imageStorage"]
+        for key in storageClaims:
+            assert (
+                agent_service_config["spec"][key]["storageClassName"] in storage_classes
```

Comment:
Is is possible that `key` will not appear in ` agent_service_config` ? if so we will just get a `ValueError` or something not descriptive right ?

Reply:
Same for the other fields accessed in objects

Reply:
I have added some, but most I think are adding a lot of verbosity but we don't gain much in safety (i.e. <obj>.metadata.name)